### PR TITLE
Dutch Language

### PR DESCRIPTION
**Pull request by adriaan1313**
I tried my best?
Is this what you mean by "send us the exported file"?
Also we don't really have a word for to shatter, so I just used the one for to break.
and Skitter, what does that even mean?
also need to fix the `\n`s and item.ttb.mysterious_book.where_am_i.page_0 doesn't really work

### CODE REVIEW

File: ttb_resources/assets/ttb/lang/nl_nl.json [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  "structure.ttb.snow_glacier": "Gletsjer",
+  "structure.ttb.snow_house": "Dorpelingenhut",
+  "structure.ttb.snow_igloo": "Sneeuw Kamp",
+  "structure.ttb.snow_spikes": "Sneeuw Kegel",
+  "structure.ttb.snowy_glass_lab": "Ondergronds Laboratorium",
+  "structure.ttb.stray_dungeon": "Verdwaalde Kerker",
+  "structure.ttb.stray_tower": "Verdwaalde Toren",
+  "text.ttb.debug.entity_removal": "Entity verwijderen: %s",
+  "text.ttb.entity.from": "%s van %s",
+  "text.ttb.entity.health_bar": "[%s] %s/%sgezondheidspunten",
```
Comment (dvitski) [inline ttb_resources/assets/ttb/lang/nl_nl.json]: `text.ttb.entity.health_bar`:
Is there a better way to write this? It shows up above an entity by default as, for example, `[Mage] 100/150HP`

**Review by dvitski — CHANGES_REQUESTED** [review]

File: ttb_resources/assets/ttb/lang/nl_nl.json (hunk 1 above)
Comment (dvitski) [reply ttb_resources/assets/ttb/lang/nl_nl.json]: Imagine this with that translation lmao:
image

Comment (adriaan1313) [reply ttb_resources/assets/ttb/lang/nl_nl.json]: let's just leave it as hp then, there is no well known-ish equivalent that is short

File: ttb_resources/assets/ttb/lang/nl_nl.json [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  "item.ttb.staff.refrigerant": "Afkoelende",
+  "item.ttb.staff.relativity": "Relativiteits",
+  "item.ttb.staff.repulsion": "Afstoting",
+  "item.ttb.staff.saturation": "Verzadiging",
+  "item.ttb.staff.shatter": "Breek",
+  "item.ttb.staff.skitter": "Skitter",
+  "item.ttb.staff.spirit": "Geest",
+  "item.ttb.staff.temporal": "Tijdelijke",
+  "item.ttb.staff.temporal.displaced_entity": "%s in tijd verplaatst.",
+  "item.ttb.staff.temporal.sped_up_time": "%s versnelde tijd",
```
Comment (dvitski) [inline ttb_resources/assets/ttb/lang/nl_nl.json]:
```suggestion
  "item.ttb.blight_staff": "Blight Staff",
  "item.ttb.captive_staff": "Captive Staff",
  "item.ttb.ether_staff": "Ether Staff",
  "item.ttb.extraction_staff": "Extraction Staff",
  "item.ttb.harvest_staff": "Harvest Staff",
  "item.ttb.harvest_staff.regress.took_lifeforce": "Niet genoeg ervaring om goed terug te gaan - je levenskracht heeft de kosten gecompenseerd",
  "item.ttb.harvest_staff.regressed_spawn_egg.lore": "Teruggegaan van volwassenheid",
  "item.ttb.infallible_staff": "Infallible Staff",
  "item.ttb.mage_staff": "Mage Staff",
  "item.ttb.netherknight_staff": "Netherknight Staff",
  "item.ttb.refrigerant_staff": "Refrigerant Staff",
  "item.ttb.relativity_staff": "Relativity Staff",
  "item.ttb.repulsion_staff": "Repulsion Staff",
  "item.ttb.saturation_staff": "Saturation Staff",
  "item.ttb.shatter_staff": "Shatter Staff",
  "item.ttb.skitter_staff": "Skitter Staff",
  "item.ttb.spirit_staff": "Spirit Staff",
  "item.ttb.temporal_staff": "Temporal Staff",
  "item.ttb.temporal_staff.sped_up_time": "%s versnelde tijd",
  "item.ttb.temporal_staff.displaced_entity": "%s in tijd verplaatst.",
```

**Review by dvitski — CHANGES_REQUESTED** [review]

File: ttb_resources/assets/ttb/lang/nl_nl.json (hunk 2 above)
Comment (adriaan1313) [reply ttb_resources/assets/ttb/lang/nl_nl.json]: I'll do that but in dutch

**Review by dvitski — APPROVED** [review]
Is this all good to go?

Edit: just noticed your edits, could we try to resolve these on Discord?

**Comment by adriaan1313** [comment]
> Edit: just noticed your edits, could we try to resolve these on Discord?
I'll just fix the newlines myself

**Comment by adriaan1313** [comment]
No, it's ok Google translate says it's Skitter in dutch as well

**Pull request merged**